src/rasta/headers/rasta_new.h

```diff
@@ -13,6 +13,7 @@ extern "C" {  // only need to export C interface if
 //TODO: check
 //#include <errno.h>
 #include "rastahandle.h"
+#include "event_system.h"
```

Comment:
Pro tip (not really relevant here): Since you only need `fd_event*` from this header, you could instead add a forward declaration for the struct and omit including the header (which improves compile time by some milliseconds here, but makes a difference in larger projects with many dependencies between headers).